src/modules/QtQuick/Item.js

```diff
@@ -54,7 +54,12 @@ registerQmlType({
       this.dom.style.position = "absolute";
     }
     this.dom.style.pointerEvents = "none";
-    this.dom.className = `${meta.object.$class}${this.id ? ` ${this.id}` : ""}`;
+    // In case the class is qualified, only use the last part for the css class
+    // name.
+    const classComponents = meta.object.$class.split(".");
+    this.dom.className
+      = `${classComponents[classComponents.length - 1]}${this.id ?
```

Comment:
If something like this does not fit into one line, it's better to just introduce a new varable and name it, for this to be more readable. Line length limit is exactly for that — it enforces making the code more readable and less complex.

`const classComponent = meta.object.$class.split(".").pop();` should be fine here.

Will fix this myself while landing.